### PR TITLE
[WIP] multi auto-complete

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/ralph/admin/widgets.py [hunk 1]
```diff
@@ -277,4 +318,11 @@ def render(self, name, value, attrs=None):
         if show_tooltip and current_object:
             context['tooltip'] = current_object.autocomplete_tooltip
```
Comment (ar4s) [inline src/ralph/admin/widgets.py:319]: remove it

File: src/ralph/admin/widgets.py [hunk 2]
```diff
@@ -189,70 +215,85 @@ def render(self, name, value, attrs=None):
...
+            'attrs': flatatt(attrs),
+            'related_url': related_url,
+            #x'input_field': input_field,
+            #x'searched_fields': sorted(
+            #x    set(list(chain(*searched_fields_tooltip.values())))
+            #x),
+            #x'searched_fields_tooltip': dict(searched_fields_tooltip),
             'show_tooltip': show_tooltip,
         })
         info = (self.rel_to._meta.app_label, self.rel_to._meta.model_name)
```
Comment (ar4s) [inline src/ralph/admin/widgets.py:299]: remove it

File: src/ralph/admin/widgets.py [hunk 3]
```diff
@@ -189,70 +215,85 @@ def render(self, name, value, attrs=None):
                 ),
                 current_app=self.admin_site.name,
             ) + self.get_url_parameters()
-            searched_fields = []
-            searched_fields_tooltip = defaultdict(list)
-            polymorphic_descendants = getattr(
-                self.rel_to, '_polymorphic_descendants', []
-            )
+            #xsearched_fields = []
```
Comment (ar4s) [inline src/ralph/admin/widgets.py:218]: remove it

File: src/ralph/admin/widgets.py [hunk 4]
```diff
@@ -189,70 +215,85 @@ def render(self, name, value, attrs=None):
...
+            #x'current_object': current_object,
+            'name': name or "",
+            'value': value,
+            'attrs': flatatt(attrs),
+            'related_url': related_url,
+            #x'input_field': input_field,
+            #x'searched_fields': sorted(
+            #x    set(list(chain(*searched_fields_tooltip.values())))
+            #x),
+            #x'searched_fields_tooltip': dict(searched_fields_tooltip),
```
Comment (ghost) [inline src/ralph/admin/widgets.py:296]: get it from l:239

**Comment by ghost** [comment]
Replaced by:
https://github.com/allegro/ralph/pull/2196

**Pull request closed without merging**